README.md

```diff
@@ -168,15 +168,14 @@ GraphQL can aggregate multiple REST requests into one query. GraphQL server orga
 - GraphQL supports queries, mutations (applying data modifications to resources), and subscriptions (receiving notifications on schema modifications).
 
 ### How does gRPC work?
+The diagram below illustrates the overall data flow for **gRPC**.
 
 <p>
   <img src="images/grpc.jpg">
 </p>
 
 RPC (Remote Procedure Call) is called “**remote**” because it enables communications between remote services when services are deployed to different servers under microservice architecture. From the user’s point of view, it acts like a local function call.
 
-The diagram below illustrates the overall data flow for **gRPC**.
```

Comment:
I think it's slightly more readable if we keep this line where it is, and move the diagram below this line. This way we have a short  intro paragraph first before diving into the picture. wdyt?

Reply:
@slam Thant makes more sense. I will update my PR